releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -2271,8 +2501,8 @@
   },
   "124503": {
     "commit": "bae83009d36fc9729482404dddc3b03ab751f08e",
-    "text": "kubeadm: during the preflight check \"CreateJob\" of \"kubeadm upgrade\", check if there are no nodes where a Pod can schedule. If there are none, show a warning and skip this preflight check. This can happen in single node clusters where the only node was drained.",
-    "markdown": "Kubeadm: during the preflight check \"CreateJob\" of \"kubeadm upgrade\", check if there are no nodes where a Pod can schedule. If there are none, show a warning and skip this preflight check. This can happen in single node clusters where the only node was drained. ([#124503](https://github.com/kubernetes/kubernetes/pull/124503), [@neolit123](https://github.com/neolit123)) [SIG Cluster Lifecycle]",
+    "text": "kubeadm: Added support during the preflight check \"CreateJob\" of \"kubeadm upgrade\" to check if there are no nodes where a Pod can be scheduled. If there are none, show a warning and skip this preflight check. This can happen in single node clusters where the only node was drained.",
+    "markdown": "Kubeadm: Added support during the preflight check \"CreateJob\" of \"kubeadm upgrade\" to check if there are no nodes where a Pod can be scheduled. If there are none, show a warning and skip this preflight check. This can happen in single node clusters where the only node was drained. ([#124503](https://github.com/kubernetes/kubernetes/pull/124503), [@neolit123](https://github.com/neolit123))",
```

Comment:
```suggestion
    "markdown": "Kubeadm: Added support during the preflight check \"CreateJob\" of \"kubeadm upgrade\" to check if there are no nodes where a Pod can be scheduled. If there are none, show a warning and skip this preflight check. This can happen in single node clusters where the only node was drained. ([#124503](https://github.com/kubernetes/kubernetes/pull/124503), [@neolit123](https://github.com/neolit123)) [SIG Cluster Lifecycle]",
```